jwst/extract_2d/grisms.py

```diff
@@ -587,8 +609,12 @@ def compute_tso_wavelength_array(slit):
     return wavelength
 
 
-def compute_tso_offset_center(input_model: ImageModel, distortion: CompoundModel) -> tuple[float, float]:
+def compute_tso_offset_center(
+    input_model: ImageModel, distortion: CompoundModel
+) -> tuple[float, float]:
     """
+    Accurately compute the offset between the source position and aperture center.
```

Comment:
```suggestion
    Compute the offset between the source position and aperture center.
```

Reply:
I don't see what harm the word 'accurately' does here, but I'm OK with removing it